control/control.openSUSE.xml

```diff
@@ -285,66 +285,117 @@ textdomain="control"
-
-        <!-- Subvolumes to be created for a Btrfs root file system -->
-        <!-- copy_on_write is true by default -->
-        <!-- The XML parser is very simplistic, thus not using attributes, but sub-elements -->
-        <subvolumes config:type="list">
-            <subvolume>
-                <path>home</path>
-            </subvolume>
-            <subvolume>
-                <path>opt</path>
-            </subvolume>
-            <subvolume>
-                <path>srv</path>
-            </subvolume>
-            <subvolume>
-                <path>tmp</path>
-            </subvolume>
-            <subvolume>
-                <path>usr/local</path>
-            </subvolume>
-            <subvolume>
-            <!-- unified var subvolume - https://lists.opensuse.org/opensuse-packaging/2017-11/msg00017.html -->
-            <path>var</path>
-                <copy_on_write config:type="boolean">false</copy_on_write>
-            </subvolume>
-
-            <!-- architecture specific subvolumes -->
-
-            <subvolume>
-                <path>boot/grub2/i386-pc</path>
-                <archs>i386,x86_64</archs>
-            </subvolume>
-            <subvolume>
-                <path>boot/grub2/x86_64-efi</path>
-                <archs>x86_64</archs>
-            </subvolume>
-            <subvolume>
-                <path>boot/grub2/powerpc-ieee1275</path>
-                <archs>ppc,!board_powernv</archs>
-            </subvolume>
-            <subvolume>
-                <path>boot/grub2/x86_64-efi</path>
-                <archs>x86_64</archs>
-            </subvolume>
-            <subvolume>
-                <path>boot/grub2/s390x-emu</path>
-                <archs>s390</archs>
-            </subvolume>
-        </subvolumes>
+        <expert_partitioner_warning config:type="boolean">false</expert_partitioner_warning>
 
+        <proposal>
+            <lvm config:type="boolean">false</lvm>
+            <proposal_settings_editable config:type="boolean">true</proposal_settings_editable>
+            <root_subvolume_read_only config:type="boolean">false</root_subvolume_read_only>
+        </proposal>
+
+        <volumes config:type="list">
+            <!-- / volume: 5 GiB - 20 GiB, *2 if snapshots are used -->
+            <volume>
+                <mount_point>/</mount_point>
+                <fs_type>btrfs</fs_type>
+
+                <desired_size config:type="disksize">10 GiB</desired_size>
+                <min_size config:type="disksize">5 GiB</min_size>
+                <max_size config:type="disksize">20 GiB</max_size>
+                <weight config:type="integer">60</weight>
+
+                <snapshots config:type="boolean">true</snapshots>
+                <snapshots_configurable config:type="boolean">true</snapshots_configurable>
+                <snapshots_percentage config:type="integer">100</snapshots_percentage>
+
+                <!-- the default subvolume "@" was requested by product management -->
```

Comment:
Did you mean "product manager" ?

Reply:
It's a historical comment, not new. See the other pr.